climpred/prediction.py

```diff
@@ -279,16 +277,22 @@ def predictability_horizon(skill,
         ph (xarray object)
     """
     if (limit is 'upper') and (not perfect_model):
-        if (p_values is None):
-            raise KeyError("""Please submit p values associated with the
-                correlation coefficients.""")
-        if (p_values.dims != skill.dims):
-            raise DimensionError("""Please submit an xarray object of the same
+        if p_values is None:
```

Comment:
In a separate PR we should pull these out to be consistent with other checks.

Reply:
I don't know exactly what you mean here. But you can open an issue to detail it more. I might remove this whole function for v1 and put in a NotImplentedError. We have it in the commit history to revisit for when we do horizons. This is a legacy function that we were using for our personal projects that isn't perfect.